cpp/perspective/src/cpp/sparse_tree.cpp

```diff
@@ -2020,6 +2014,28 @@ t_stree::get_aggregate(t_index idx, t_index aggnum) const {
     return extract_aggregate(m_aggspecs[aggnum], c, agg_ridx, agg_pridx);
 }
 
+t_tscalar
+t_stree::get_aggregate_median(std::vector<t_tscalar>& values) const {
+    int size = values.size();
+    bool is_even_size = size % 2 == 0;
+
+    if (is_even_size && values[0].is_numeric()){
```

Comment:
See review summary - `is_numeric()` case here should be exclusively `float` column types via  `is_floating_point()`.

Reply:
Done!